File: src/main/java/no/nav/fo/veilarbregistrering/config/PingFilter.java
package no.nav.fo.veilarbregistrering.config;

import org.springframework.http.HttpStatus;

import javax.servlet.FilterConfig;
import javax.servlet.*;
import javax.servlet.http.HttpServletResponse;

public class PingFilter implements Filter {

    @Override
    public void init(FilterConfig filterConfig) {}

    @Override
    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) {
        HttpServletResponse httpServletResponse = (HttpServletResponse) servletResponse;
        httpServletResponse.setStatus(HttpStatus.OK.value());
    }

    @Override
    public void destroy() {}

}

File: src/main/java/no/nav/fo/veilarbregistrering/config/filters/PingFilter.kt
package no.nav.fo.veilarbregistrering.config.filters

import org.springframework.http.HttpStatus
import javax.servlet.*
import javax.servlet.FilterConfig
import javax.servlet.http.HttpServletResponse

class PingFilter : Filter {
    override fun init(filterConfig: FilterConfig) {}
    override fun doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain) {
        val httpServletResponse = servletResponse as HttpServletResponse
        httpServletResponse.status = HttpStatus.OK.value()
    }

    override fun destroy() {}
}